strings/src/main/res/values/strings.xml

```diff
@@ -1214,4 +1214,15 @@
     <string name="edit_form">Edit</string>
     <string name="view_form">View</string>
     <string name="close_snackbar">Close snackbar</string>
+
+    <!--
+    ##############################################
+    # Forms ready to send banner
+    ##############################################
+    -->
+    <string name="last_form_sent_seconds_ago">Last form sent: %d second(s) ago</string>
```

Comment:
I could have used [Quantity strings](https://developer.android.com/guide/topics/resources/string-resource#Plurals) instead of adding `(s)` but we usually use the latter because it's easier. Supporting so many languages and keeping such strings translated correctly would be difficult.

Reply:
In this case I think we should go for the quantity string. Many workflows involve looking at this screen a LOT. I think we want it to look as clean as we can.

Reply:
I agree! I took a screengrab below – it isn't great for readability.

Reply:
Ok, I've improved that.